ledger/ledger-api-client/src/test/suite/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlowTest.scala

```diff
@@ -133,26 +134,6 @@ class CommandTrackerFlowTest
 
   "Command tracking flow" when {
 
-    "no configuration is available" should {
-      "fail immediately" in {
-        val Handle(submissions, results, _, _) =
-          runCommandTrackingFlow(allSubmissionsSuccessful, maxDeduplicationTime = None)
-
-        submissions.sendNext(submission)
-
-        results.requestNext().value shouldEqual Left(
-          NotOkResponse(
-            commandId,
-            Status.of(
-              Code.UNAVAILABLE.value,
-              "The ledger configuration is not available.",
-              Seq.empty,
-            ),
-          )
-        )
-      }
-    }
-
```

Comment:
This shouldn't be possible anymore, right?

Reply:
Correct; you can see this in the change in types.